test/test-trace-header-context.js

```diff
@@ -25,14 +25,14 @@ describe('test-trace-header-context', function() {
   var agent;
   var express;
   before(function() {
-    agent = require('..').start({ samplingRate: 0 }).private_();
+    agent = require('..').start({ samplingRate: 0 });
     express = require('./hooks/fixtures/express4');
   });
 
   afterEach(function() {
-    // TODO: Investigate why this is needed
+    // TODO: Investigate why this is needed on Node v0.12
```

Comment:
Maybe we could expose a `common.resetNamespace` which does a `destroy` -> `create` on the agent namespace to avoid this access to `private_`?